layouts/partials/navigation.html

```diff
@@ -26,14 +26,15 @@
     <button class="navbar-toggler border-0" type="button" data-toggle="collapse" data-target="#navigation"
       aria-controls="navigation" aria-expanded="false" aria-label="Toggle navigation">
       <span class="navbar-toggler-icon"></span>
+        <span class="close-icon py-1"><b>X</b></span>
```

Comment:
I'm curious. This change ended up showing me the "X" even when the menu was already closed. The existing toggler does not show. Is this supposed to happen?

Reply:
It should surely be a small code bug, I'll add some commits to this pull request fir the changes implementation.😀

Reply:
@yokwejuste Sounds good. Were you able to figure it out? If not, I suggest we revert this part of the change and instead focus on the original change for the `:hover` effect only.